### DISCUSSION TITLE
How do I see what the latest stable version is from the CLI?

### DISCUSSION THREAD

**Discussion by qcn** [yarnpkg/berry#3835] [Q&A] [answered]
I know I can run `yarn set version stable` to update yarn inside my project to the latest stable version. Is there a command I can run to find out what that version _is_ before I update/install it?

**Comment by paul-soporan** [comment, marked as answer]
We don't currently have a command for it, but for now you can run `curl -s https://repo.yarnpkg.com/tags | jq .latest.stable` to resolve the `stable` tag just like `yarn set version` does.

**Reply by qcn** [reply]
Awesome, thank you so much! I should have thought to look at what the command does :)
